pidev/src/main/java/led/lightstrip/scrum5/pidev/PidevApplication.java

```diff
@@ -1,13 +1,30 @@
 package led.lightstrip.scrum5.pidev;
 
+import models.Device;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class PidevApplication {
 
+
+
+
 	public static void main(String[] args) {
 		SpringApplication.run(PidevApplication.class, args);
+
+
+		// testing factory
+		Device myDevice = new factory().createDevice("neopixel");
```

Comment:
Can change this to use parameter passed into factory. In the constructor createDevice() can happen.